app/presenters/campaigns/europe_at_work/index.rb

```diff
@@ -22,15 +23,15 @@ def include_nav_searchbar
       end
 
       def page_content_heading
-        t('title')
-      end
-
-      def contentful_entry
-        contentful.entries(content_type: 'staticPage', include: 2, 'fields.identifier' => 'europe-at-work').first
+        static_page.fields[:headline]
       end
 
       protected
 
+      def static_page
+        contentful_entry(identifier: 'europe-at-work')
```

Comment:
Memoize this else you end up making multiple identical calls to Contentful for the one page:
```ruby
@static_page ||= contentful_entry(identifier: 'europe-at-work')
```